src/Hl7.Fhir.Shared.Tests/Rest/FhirClientTests.cs

```diff
@@ -895,6 +896,21 @@ public async T.Task TestMultipleMessageHandlersInFhirClient()
             Assert.IsNotNull(testDegatingHandler.LastRequest);
             Assert.IsNotNull(testMessageHandler.LastResponse);
         }
+        
+        
+        [TestMethod]
+        [TestCategory("IntegrationTest"), TestCategory("FhirClient")]
```

Comment:
Did you test the normal behaviour - have you run some of the other Fhirclient tests to see whether we had regressions?